src/com/CommunicateMesh.cpp

```diff
@@ -48,17 +53,25 @@ void CommunicateMesh::sendMesh(
     _communication->sendRange(globalIDs, rankReceiver);
   }
 
+  _communication->send(mesh.hasConnectivity(), rankReceiver);
+  if (!mesh.hasConnectivity()) {
+    PRECICE_DEBUG("No connectivity to send");
+    return;
+  }
+
+  // we need to send the vertexIDs first such that the right connectivity can be created later
+  // contrary to the normal sendMesh, this variant must also work for adding delta meshes
```

Comment:
Do you know what is meant by "normal sendMesh" here? I have the feeling this is some historic left-over comment.

Reply:
I don't know and cannot quite figure out what this means.
Maybe this used to be part of the broadcast code?

Reply:
Yes, could very well be.